### ISSUE TITLE
BUG: save loop in trainer not set up to sample from model without text conditioning

### ISSUE THREAD

**Issue by Ljackson-su** [Oxford-Dynamics-Repository/mindglass#22]
Line 264 of trainer.py throws an error when sampling without text embeding.

Need to add:
`if not self.has_label:
      cond = None`
Above 264

**Comment by oxjohanndiep** [comment]
Good catch, I assume you made that change already when you guys were producing the un-conditioned results with the pianist?

**Comment by Ljackson-su** [comment]
Yes, I made the edit. Just need to think about what we want the inference conditioning to be during training (we have discussed this elsewhere!)
